android/app/src/main/kotlin/com/jabook/app/jabook/compose/data/network/DynamicBaseUrlInterceptor.kt

```
@@ -209,32 +209,32 @@ public class DynamicBaseUrlInterceptor
                         try {
                             val retryStartTime = System.currentTimeMillis()
                             val retryResponse = chain.proceed(retryRequest)
                             val retryDuration = System.currentTimeMillis() - retryStartTime
                             if (retryResponse.isSuccessful) {
                                 logger.i {
-                                    "✅ Retry succeeded: ${retryResponse.code} (${retryDuration}ms) with mirror $newMirror"
+                                    "Retry succeeded: ${retryResponse.code} (${retryDuration}ms) with mirror $newMirror"
                                 }
                             }
                             return retryResponse
                         } catch (retryException: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)